models/cnn.py

```diff
@@ -0,0 +1,35 @@
+import tensorflow as tf
+
+from tensorflow.keras import Model
+from tensorflow.keras.layers import InputLayer, Conv2D, Flatten, Dense, AveragePooling2D, MaxPool2D
+
+class ConvModel(Model):
+    def __init__(self,name="conv_basic"):
+        super(ConvModel,self).__init__()
+
+        self.input_layer = InputLayer((128,430,1), name=f"{name}_input")
```

Comment:
We might want to think about using some parameters for the input dimensions. But not urgent.